frontend/src/components/App/LoginForm.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react'
+
+class LoginForm extends Component {
+  static propTypes = {
+    loginFormAuthToken: PropTypes.string
+  }
+
+  render () {
+    return <form className="loginAnywhere" id="new_user" action="/login" acceptCharset="UTF-8" method="post">
```

Comment:
It's just a reference to how this particular form enables you to log in from any page. rather than having to go to the specific login page to do it